src/packcc.c

```diff
@@ -1432,95 +1432,137 @@ static void verify_captures(context_t *ctx, node_t *node, node_const_array_t *ca
     }
 }
 
-static void dump_node(context_t *ctx, const node_t *node) {
+void dump_escaped(const char* s) {
```

Comment:
It's better to add a `static` modifier to make this function hidden under file-scope.

Reply:
Right, I keep forgeting this is not C++ :slightly_smiling_face: Fixed now.